### PR TITLE
problem fixed

### LINKED ISSUE #1: Reason for getting Nan Values and other improvements
I notice that you are getting Nan errors while training your linear regression model. It is due to a misplaced value in the train.csv file -> [here](https://github.com/suzil/regression-in-pytorch/blob/master/train.csv#L215). As you can observe there is no y value for the input x value and due to which the pandas dataframe contains a NaN value. 

Upon removing this nan Value, you'll still face training problem because you have not mean-normalized the input data. Your model will take forever to reach convergence. One very cheap work-around is to initialize the model at `w = 1; b = 0` because we already know the line that should give the best result. But, this is not possible in very high dimensional data. Normalizing the data (which I mentioned earlier) is the way to go.

### PR DESCRIPTION
**Pull request by akanimax**
#1 addressing the problems mentioned in this issue.

### CODE REVIEW

**Comment by akanimax** [comment]
I have removed the dependency on the deprecated `torch.Variable` module. In the current stable version of pytorch, A tensor is itself compatible with the `torch.autograd` for performing backpropagation. 

Cheers!

**Comment by suzil** [comment]
Super, thanks :slightly_smiling_face: !
